components/content/src/ser.rs

```diff
@@ -162,6 +162,9 @@ pub struct SerializingSection<'a> {
     backlinks: Vec<BackLink<'a>>,
     generate_feeds: bool,
     transparent: bool,
+    paginated: bool,
```

Comment:
No need for that one, let's match what is in the frontmatter

Reply:
I just removed the unecessary `paginated` attribute and retrieved raw values from the frontmatter.

Adapted testsite: [testsite.zip](https://github.com/user-attachments/files/17345078/testsite.zip)